iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="backdrop" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" role="none" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
Thanks for the pointer!

I think the nicest option would be for the `img` role to be used only when the actual image is hidden, and to use the `<img>` itself whenever it is available. That way you can use the native `alt` attribute of the image as its accessible alternative as well as getting the additional benefit of `alt` if the image fails to load, as well as not needing to leak the role and label out into the host DOM.

If that's not feasible, then the solution you have is an ok alternative.